### PR TITLE
chore: update changelog

### PR DESCRIPTION
**Pull request by gigeos**
# General summary

# Screenshots

# Check list

-   [ ] (if has style) Add `need: review-integration` label
-   [x] (if has textual documentation) Add `need: review-doc` label
-   [ ] (if has code) Add `need: review-frontend` label
-   [ ] (if has react) Check through the [react dev check list]
-   [ ] (if fix or feature) Add `need: test` label

Check out the [contribution guide] for general guidelines for submissions to the design system.

[contribution guide]: https://github.com/lumapps/design-system/blob/master/CONTRIBUTING.md
[react dev check list]: https://github.com/lumapps/design-system/blob/master/CONTRIBUTING.md#react-developpment-check-list

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.21.7][] - 2020-02-14
```
Comment (gcornut) [inline CHANGELOG.md]: These versions are auto generated with yarn release. We change the changelog under the Unreleased section

Comment (gigeos) [reply CHANGELOG.md]: So I just remove `## [0.21.7] ......` ?

Comment (gcornut) [reply CHANGELOG.md]: Yep

File: CHANGELOG.md [hunk 2]
```diff
@@ -327,3 +331,5 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 [0.21.5]: https://github.com/lumapps/design-system/tree/v0.21.5
 [unreleased]: https://github.com/lumapps/design-system/compare/v0.21.6...HEAD
 [0.21.6]: https://github.com/lumapps/design-system/tree/v0.21.6
+[unreleased]: https://github.com/lumapps/design-system/compare/v0.21.7...HEAD
```
Comment (matmkian) [inline CHANGELOG.md]: To remove also @gcornut ?

File: CHANGELOG.md [hunk 3]
```diff
@@ -327,3 +331,5 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+[0.21.7]: https://github.com/lumapps/design-system/tree/v0.21.7
```
Comment (gcornut) [inline CHANGELOG.md]: Both of these lines should also be removed :)

**Pull request merged**